source/JsMaterialX/test/testHelpers.js

```diff
@@ -1,20 +1,12 @@
 import Module from './_build/JsMaterialX.js';
 // import Module from './_build/JsMaterialX-1.37.1.js'; // uncomment to load the versioned JsMaterialX file
-import regeneratorRuntime from 'regenerator-runtime'; // This is required for the async/awaits
+import 'regenerator-runtime'; // This is required for the async/awaits
 var fs = require('fs');
 var path = require('path');
 /**
  * Returns a promise that resolves the MaterialX namespace
  */
-export function initMaterialX() {
-    return new Promise(function (resolve) {
-        // Note: Module is not a promise.
-        // The then function is defined by emscripten.
-        Module().then((module) => {
-            resolve(module);
-        });
-    });
-}
+export const initMaterialX = Module;
```

Comment:
If this 'helper method' doesn't do anything anymore, we can also get rid of it and import `Module` directly in all test files.

Reply:
It imports that weird `regenerator-runtime`